src/autoyast-rnc/auth-client.rnc

```diff
@@ -1,301 +0,0 @@
-default namespace = "http://www.suse.com/1.0/yast2ns"
```

Comment:
No luck, the XML serialiser seriously cannot cope with the structure:

https://bugzilla.suse.com/show_bug.cgi?id=973341
https://bugzilla.suse.com/show_bug.cgi?id=973342